README.md

```diff
@@ -95,3 +95,17 @@ To help developers track changes in files easily, it is best if you commit V2 fi
 ## Detailed documentation
 
 - [How to archive a Smart Answer](doc/archiving.md)
+
+## Deploying to Heroku
+
+    heroku apps:create
```

Comment:
Heroku(Amazon) has servers in Europe, so `heroku apps:create --region eu` would have lower latency